automatic/windows10-media-creation-tool/README.md

```diff
@@ -0,0 +1,7 @@
+# <img src="https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/tree/master/icons/windows10-media-creation-tool.png" width="48" height="48"/> [windows10-media-creation-tool](https://chocolatey.org/packages/windows10-media-creation-tool)
```

Comment:
Readme should be sorted out. Some of text belongs to Notes. Features if possible.

Reply:
added notes and features